modules/battle.lua

```diff
@@ -82,16 +100,22 @@ performBattle = function()
 	end
 	
 	if _card.canClickNpCards() then
-		_card.clickNpCards()
+		NPsClicked = _card.clickNpCards()
 	end
 	
-	_card.clickCommandCards()
+	_card.clickCommandCards(5)
 
 	if UnstableFastSkipDeadAnimation == 1 then
 		skipDeathAnimation()
 	end
-
-	wait(2)
+	
+	_card.resetCommandCards()
+	
+	if NPsClicked then
+		wait(25)
```

Comment:
why this hella long delay?

Reply:
NP animations can be really long, but this isn't the longest delay that could be used for them. I made it slightly shorter than the average I believe.